_data/glossary/labels.yaml

```diff
@@ -0,0 +1,9 @@
+id: labels
+name: Labels
+full-link: /docs/concepts/overview/working-with-objects/labels
+tags:
+- fundamental
+short-description: >
+  Used to tag objects with identifying attributes that are meaningful and relevant to users.
+long-description: >
+  Labels are key/value pairs that are attached to objects, such as pods. They can be used to organize and to select subsets of objects.
```

Comment:
s/pods/Pods

Reply:
remove comma after `objects` in first sentence.
Also (and I struggle with this one too in intro content), "They can be used" makes them sound optional, which they are programmatically, but you pretty much need labels to do any kind of selection, and you're always needing to select things. 
Maybe s/can be used/are used ? 
But I wonder whether a little more substance to the long description might be helpful too.